src/notation/internal/notationinteraction.h

```diff
@@ -256,14 +280,17 @@ class NotationInteraction : public INotationInteraction, public async::Asyncable
 
     bool needEndTextEditing(const std::vector<EngravingItem*>& newSelectedElements) const;
 
-    void updateGripEdit(const std::vector<EngravingItem*>& elements);
+    void updateGripEdit();
     void resetGripEdit();
 
     void resetStretch();
     void resetTextStyleOverrides();
     void resetBeamMode();
     void resetShapesAndPosition();
 
+    template<typename P>
+    void execute(void (Ms::Score::* function)(P), P param);
```

Comment:
"catch-all" function to allow executing Score:: methods without INotationInteraction needing to get too bloated